### DISCUSSION TITLE
Questions about the output files in Cardinal turorials

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#27254] [Q&A Modules: General]
@GiudGiud @milljm Hello everyone! When running the standalone example, I noticed that in addition to turbPipe.f, Cardinal also outputs avgturbPipe.f, rm2turbPipe.f, and rmsturbPipe.f. I would like to know what these additional output files represent and how they are generated through which settings.

**Comment by GiudGiud** [comment]
Hello

this is best asked in the Cardinal user forum

guillaume

**Reply by Ethan-xj** [reply]
OK. Thank you Giud
